doc/lua_api.txt

```diff
@@ -6946,13 +6946,16 @@ object you are working with still exists.
             (default: `true`)
         * `texture`: A regular texture for the sun. Setting to `""`
             will re-enable the mesh sun. (default: "sun.png", if it exists)
+            At sunrise, the texture appears non-rotated.
```

Comment:
compared to what?

Reply:
Compared to the screen.
This assumes the current FPS-style camera.

Reply:
Description adjusted for clarity.